com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -209,6 +208,42 @@ public Animator Animator
             }
         }
 
+        [Tooltip("When enabled, the NetworkAnimator will operate in server authoritative mode.")]
+        [SerializeField]
+        private bool m_IsServerAuthoritative;
+
+        /// <summary>
+        /// Server-Side Only:
+        /// Can be used to switch between server and owner authoritative modes during runtime.
+        /// Note: Only when the associated NetworkObject is spawned.
+        /// </summary>
+        public bool IsServerAuthoritative
```

Comment:
to keep this API similar to `NetworkTransform`'s, I think we should go for something similar to (protected) `NetworkTransform::OnIsServerAuthoritatitive()` & (internal) `NetworkTransform::IsServerAuthoritatitive()`